tools/apiview/parsers/CONTRIBUTING.md

```diff
@@ -45,66 +57,32 @@ Each tree node has top tokens which should be used to capture the main tokens on
-- `Tags` : Use this for opt in or opt out boolean properties.The currently supported tags are 
-  - `SkippDiff`  Indicate that a token should not be used in computatation of diff.
-  - `Deprecated` Mark a token as deprecated
-- `Properties` : Properties of the token.
-  - `GroupId` : `doc` to group consecutive comment tokens as documentation.
-  - `NavigateToId` Id for navigating to where the node where the token is defined. Should match the definitionId of the referenced node.
-- `RenderClasses` : Add css classes for how the tokens will be rendred. Classes currently being used are `text` `keyword` `punc` `tname` `mname` `literal` `sliteral` `comment` Feel free to add your own custom class. Whatever custom classes you use please provide us the appriopriate css for the class so we can update APIView.
-
-
-Assign the final parsed value to a `List<APITreeNode> APIForest` property of the `CodeFile`
-
-## Serilizations
-
-Serialize the generated code file to JSON them compress the file using Gzip compression. Try to make the json as small as possible by ignoring null values and empty collections.
+Serialize the generated code file to JSON and then compress the file using Gzip compression. Try to make the json as small as possible by ignoring null values and empty collections.
+Don't worry about indentation that will be handled by the tree structure. In the case you want to have indentation between the tokens then use `TabSpace` token kind.
 
 ## How to handle commons Scenarios
-- TEXT, KEYWORD, COMMENT : Add `text`, `keyword`, `comment` to RenderClasses of the token
-- NEW_LINE : Create a token with `Kind = LineBreak`
-- WHITE_SPACE :  Create token with `Kind = NonBreakingSpace`
-- PUNCTUATION : Create a token with `Kind = Content` and the `Value = the punctuation`
+
+- TEXT, KEYWORD, COMMENT : Use `text`, `keyword`, `comment` to property `RenderClasses` of `StructuredToken`.
+- NEW_LINE : Create a token with `Kind = LineBreak`.
+- WHITE_SPACE :  Create token with `Kind = NoneBreakingSpace`.
+- PUNCTUATION : Create a token with `Kind = Content` and the `Value = <the punctuation>`.
 - DOCUMENTATION : Add `GroupId = doc` in the properties of the token. This identifies a range of consecutive tokens as belonging to a group.
-- SKIP_DIFF :  Add `SkipDiff` to the Tag to indicate that node or token should not be included in diff computation
+- SKIP_DIFF :  Add `SkipDiff` to the Tag to indicate that the node or token should not be included in diff computation.
 - LINE_ID_MARKER : You can add a empty token. `Kind = Content` and `Value = ""` then give it an `Id` to make it commentable.
 - EXTERNAL_LINK : Create a single token set `Kind = Url`, `Value = link` then add the link text as a properties `LinkText`;
 - Common Tags: `Deprecated`, `Hidden`, `HideFromNav`, `SkipDiff`
 - Cross Language Id: Use `CrossLangId` as key with value in the node properties.
```

Comment:
These should probably be little subsections, and show the necessary JSON payload. That's the only cross-language representation there is.

Reply:
We can add more information about cross language when the feature is fully implemented